### ISSUE TITLE
EdgeChromiumDriverManager and IEDriverManager initialization fails

### ISSUE THREAD

**Issue by jammer87** [SergeyPirogov/webdriver_manager#269]
I'm getting this error with version 3.5.2. Reverting to 3.5.1 resolves the issue. I do not have any WDM_ environment variables set. EdgeChromiumDriverManager exception shown in the text below, same thing happens with IEDriverManager.

```text
>>> from webdriver_manager.microsoft import EdgeChromiumDriverManager
>>> EdgeChromiumDriverManager()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\xxxxx\.virtualenvs\virtual-env-location-gPpqDgo6\lib\site-packages\webdriver_manager\microsoft.py", line 50, in __init__
    super().__init__(path, log_level, print_first_line, cache_valid_range)
  File "C:\Users\xxxxx\.virtualenvs\virtual-env-location-gPpqDgo6\lib\site-packages\webdriver_manager\manager.py", line 13, in __init__
    log("====== WebDriver manager ======", level=log_level)
  File "C:\Users\xxxxx\.virtualenvs\virtual-env-location-gPpqDgo6\lib\site-packages\webdriver_manager\logger.py", line 25, in log
    _init_logger(level, name, first_line, formatter)
  File "C:\Users\xxxxx\.virtualenvs\virtual-env-location-gPpqDgo6\lib\site-packages\webdriver_manager\logger.py", line 19, in _init_logger
    _logger.setLevel(level)
  File "C:\Program Files\Python38\lib\logging\__init__.py", line 1421, in setLevel
    self.level = _checkLevel(level)
  File "C:\Program Files\Python38\lib\logging\__init__.py", line 201, in _checkLevel
    raise TypeError("Level not an integer or a valid string: %r" % level)
TypeError: Level not an integer or a valid string: None
```

**Comment by natarajan0007** [comment]
Same issure for me also . Downgrading to 3.5.1 version works fine.

**Comment by TiredFingers** [comment]
Fast fix is just pass log_level to `EdgeChromiumDriverManager(log_level='INFO')`

**Comment by TiredFingers** [comment]
fast fix is `EdgeChromiumDriverManager(log_level=1)`

**Comment by Yumin2019** [comment]
Thanks. This was really helpful.

**Comment by SviatoslavBordovski** [comment]
> fast fix is `EdgeChromiumDriverManager(log_level=20)`

thanks, that has fixed the test, what does mean that 'log_level=20'?

**Comment by SviatoslavBordovski** [comment]
@SergeyPirogov фиксать есть силы или побудет opened и закроешь? :)

**Comment by SergeyPirogov** [comment]
Если можешь чини.

**Comment by aleksandr-kotlyar** [comment]
Fix is already at master branch.

**Comment by aleksandr-kotlyar** [comment]
@jammer87 thank you for the report and patience! Fix is already in [3.5.3](https://pypi.org/project/webdriver-manager/3.5.3/). Check it out!
